google/cloud/storage/client.py

```diff
@@ -836,6 +841,126 @@ def get_hmac_key_metadata(
         metadata.reload(timeout=timeout)  # raises NotFound for missing key
         return metadata
 
+    def get_signed_policy_v4(
+        self,
+        bucket_name,
+        blob_name,
+        conditions,
+        expiration,
+        fields=None,
+        virtual_hosted_style=False,
+        bucket_bound_hostname=None,
+        scheme=None,
```

Comment:
Missing some support for additional credential flows such as SignBlob API introduced into signedUrls in #9889.

Reply:
Done